packages/itwin/grouping-mapping-widget/src/components/Properties/GroupProperties/GroupPropertyAction.tsx

```diff
@@ -279,6 +280,19 @@ export const GroupPropertyAction = ({
     }
   }, [searchInput, setSearched, clearSearch]);
 
+  const getLocalizedStringPresentation = useCallback((input: string): string => {
```

Comment:
It looks like this function could be used anywhere where presentation is used. Maybe move it to some kind of utils?

Reply:
This is the only place this can be used as of now, because this is rendering the descriptor data directly. But I do like the idea of moving it somewhere else. I moved it.